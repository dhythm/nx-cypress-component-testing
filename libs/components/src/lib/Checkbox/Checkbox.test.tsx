import renderer from 'react-test-renderer';
import { Checkbox } from './';

describe('Checkbox', () => {
  it('Snap Shot', () => {
    const component = renderer.create(<Checkbox label="confirm" checked />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
