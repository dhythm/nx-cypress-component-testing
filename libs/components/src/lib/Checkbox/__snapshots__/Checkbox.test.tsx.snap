// Vitest Snapshot v1

exports[`Checkbox > Snap Shot 1`] = `
[
  <label
    className="chakra-checkbox css-1577qb8"
    data-checked=""
    onClick={[Function]}
  >
    <input
      checked={true}
      className="chakra-checkbox__input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      style={
        {
          "border": "0",
          "clip": "rect(0, 0, 0, 0)",
          "height": "1px",
          "margin": "-1px",
          "overflow": "hidden",
          "padding": "0",
          "position": "absolute",
          "whiteSpace": "nowrap",
          "width": "1px",
        }
      }
      type="checkbox"
    />
    <span
      aria-hidden={true}
      className="chakra-checkbox__control css-1ydjfm6"
      data-checked=""
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
    >
      <div
        className="css-0"
        style={
          {
            "alignItems": "center",
            "display": "flex",
            "height": "100%",
            "justifyContent": "center",
          }
        }
      >
        <svg
          className="css-5jdg10"
          style={
            {
              "fill": "none",
              "stroke": "currentColor",
              "strokeDasharray": 16,
              "strokeWidth": 2,
            }
          }
          viewBox="0 0 12 10"
        >
          <polyline
            points="1.5 6 4.5 9 10.5 1"
          />
        </svg>
      </div>
    </span>
    <span
      className="chakra-checkbox__label css-6x44c9"
      data-checked=""
      onMouseDown={[Function]}
      onTouchStart={[Function]}
    >
      confirm
    </span>
  </label>,
  <span />,
]
`;
